evap/evaluation/tests/tools.py

```diff
@@ -273,3 +257,75 @@ def assert_no_database_modifications(*args, **kwargs):
             lower_sql = query["sql"].lower()
             if not any(lower_sql.startswith(prefix) for prefix in allowed_prefixes):
                 raise AssertionError("Unexpected modifying query found: " + query["sql"])
+
+
+@tag("live-server")
+class LiveServerTest(SeleniumTestCase):
+    external_host = os.environ.get("TEST_HOST", "") or None
+    browser = "firefox"
+    selenium_hub = None
+    headless = True
+    window_size = (1920, 3080)
```

Comment:
3080 intended? Or should it be 1080?

Reply:
yes, the height should be large enough so that there is no scrolling needed. Maybe a comment?

Reply:
Yeah, I'd add a comment, since 1080->3080 could easily be a typo. Note that we're never guaranteed that we don't need any scrolling, the result pages can easily be dozens of screens long. But I see that something around 4k makes our lives easier here

Reply:
I am also fine with odd dimensions (and a comment) for now. We should revisit scrolling at some point though